File: helper/src/main/java/cc/xiaobaicz/helper/error/ErrorCallback.java
package cc.xiaobaicz.helper.error;

/**
 * @author BC
 * @date 2019/9/20
 */
public interface ErrorCallback {

    /**
     * 异常通知
     * @param t
     * @param throwable 异常信息
     */
    void callback(Thread t, Throwable throwable);

}

File: helper/src/main/java/cc/xiaobaicz/helper/error/ErrorCallback.kt
package cc.xiaobaicz.helper.error

/**
 * @author BC
 * @date 2019/9/20
 */
fun interface ErrorCallback {
    /**
     * 异常通知
     * @param t
     * @param throwable 异常信息
     */
    fun callback(t: Thread, throwable: Throwable)
}